zebra-network/src/meta_addr.rs

```diff
@@ -703,15 +739,34 @@ impl MetaAddrChange {
             } => Some(*untrusted_last_seen),
             NewAlternate { .. } => None,
             // We know that our local listener is available
-            NewLocal { .. } => Some(DateTime32::now()),
+            NewLocal { .. } => Some(now),
             UpdateAttempt { .. } => None,
             UpdateResponded { .. } => None,
             UpdateFailed { .. } => None,
         }
     }
 
+    // # Concurrency
+    //
+    // We assign a time to each change when it is applied to the address book by either the
+    // address book updater or candidate set tasks. This is the time that the change was received
+    // from the updater channel, rather than the time that the message was read from the peer
+    // connection.
```

Comment:
This was surprising to me: address book updates get given a time when they are received from the channel, rather than when they actually happen.

But we don't need to fix this design issue to fix the audit issue.